src/extension/decorations/closing_labels_decorations.ts

```diff
@@ -57,27 +65,31 @@ export class ClosingLabelsDecorations implements vs.Disposable {
 			// Get the end of the line where we'll show the labels.
 			const endOfLine = this.activeEditor.document.lineAt(finalCharacterPosition).range.end;
 
+			const closingLabelsPrefix = config.closingLabelsPrefix;
+
 			const existingDecorationForLine = decorations[endOfLine.line];
 			if (existingDecorationForLine) {
-				existingDecorationForLine.renderOptions.after.contentText = " // " + r.label + " " + existingDecorationForLine.renderOptions.after.contentText;
+				existingDecorationForLine.renderOptions.after.contentText = closingLabelsPrefix + r.label + " " + existingDecorationForLine.renderOptions.after.contentText;
 			} else {
 				const dec = {
 					range: new vs.Range(this.activeEditor.document.positionAt(r.offset), endOfLine),
-					renderOptions: { after: { contentText: " // " + r.label } },
+					renderOptions: { after: { contentText: closingLabelsPrefix + r.label } },
 				};
 				decorations[endOfLine.line] = dec;
 			}
 		}
 
-		this.activeEditor.setDecorations(this.decorationType, Object.keys(decorations).map((k) => parseInt(k, 10)).map((k) => decorations[k]));
+		this.recreateDecorationType();
+
+		this.activeEditor.setDecorations(this.decorationType!, Object.keys(decorations).map((k) => parseInt(k, 10)).map((k) => decorations[k]));
 	}
 
 	private setTrackingFile(editor: vs.TextEditor | undefined) {
 		if (!editor || !isAnalyzable(editor.document))
 			return;
 
 		if (editor !== this.activeEditor)
-			this.activeEditor?.setDecorations(this.decorationType, []);
+			this.activeEditor?.setDecorations(this.decorationType!, []);
```

Comment:
I think it's be nice to avoid the `!`s here.. perhaps by making sure it's first set up in the constructor and removing the null option from it.